syntax = "proto3";

service Showtime {
    rpc GetSchedule(Empty) returns (stream Time) {}
    rpc GetMoviesByDate(Date) returns (Movies) {}
}

message Date {
    string date = 1;
}

message Movies {
    repeated string movies = 1;
}

message Time {
    string date = 1;
    repeated string movies = 2;
}

message Empty {
}